python-rhsm/src/rhsm/connection.py

```diff
@@ -1187,7 +1187,7 @@ def checkin(self, consumerId, checkin_date=None):
 
         return self.conn.request_put(method)
 
-    def getPoolsList(self, consumer=None, listAll=False, active_on=None, owner=None, filter_string=None):
+    def getPoolsList(self, consumer=None, listAll=False, future=None, active_on=None, owner=None, filter_string=None):
```

Comment:
I'm being a little pedantic here, but can you please move `future` to the last parameter? Ideally users will just use them as kwargs, but if they don't I'd rather not accidentally break someone's code...